src/Tribe/Main.php

```diff
@@ -2171,6 +2172,39 @@ public function loadStyle() {
 			}
 		}
 
+		/**
+		 * Before underscore is loaded to the FE we add two scripts on before and one after to prevent underscore from
+		 * taking place on the global namespace if lodash is present.
+		 *
+ 		 * @since TBD
+ 		 *
+ 		 * @param string $tag The <script> tag for the enqueued script.
+		 * @param string $handle The script's registered handle.
+		 * @return string The <script> tag.
+		 */
+		public function prevent_underscore_conflict( $tag, $handle ) {
+
+			if ( is_admin() ) {
+				return $tag;
+			}
+
+			if ( 'underscore' === $handle ) {
+				$path = plugin_dir_path( TRIBE_EVENTS_FILE ) . 'src/resources/js/';
+				$dir = plugin_dir_url( TRIBE_EVENTS_FILE ) . 'src/resources/js/';
+
+				$before = file_exists( $path . 'underscore-before.js' ) ? file_get_contents( $dir . 'underscore-before.js' ) : '';
+				if ( $before ) {
+					$tag = "<script type='text/javascript'>$before</script>\n" . $tag;
```

Comment:
If the Underscore is not async we can include it like a normal JS file, no need to make it inline.

Reply:
In this particular case we have to, as we don't know what script is going to be loaded before `underscore`, for the one after works but the one before we need to make sure is done before `underscore` is attached to the global namespace.

Reply:
What I mean was to add a `<script src="">` these scripts instead of inline including them. Manually include those but avoid reading the file via the backend.